src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingSendRequestNewTrainingProviderTests.cs

```diff
@@ -146,5 +156,24 @@ public void VerifyRedirectToError(IActionResult actionResult)
             Assert.AreEqual("Error", redirectResult.ControllerName);
             Assert.AreEqual("Error", redirectResult.ActionName);
         }
+
+        public void VerifyProviderAddDetailsIsSetToTrue(IActionResult result)
```

Comment:
I was looking for another example fo this and found the following on WhenCallingMessage.cs, the following looks a bit more concise and simple, but happy to discuss

   var redirect = result.VerifyReturnsRedirectToActionResult();
            Assert.AreEqual(redirect.ActionName, "Finished");
            Assert.AreEqual(redirect.RouteValues["AccountHashedId"], model.AccountHashedId);
            Assert.AreEqual(redirect.RouteValues["CohortReference"], createCohortResponse.CohortReference);